data.sql

```diff
@@ -12,4 +12,16 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
       (8, 'Angemon', '2005-6-12', 1, true, 45),
       (9, 'Boarmon', '2005-6-07', 7, true, 20.4),
       (10, 'Blossom', '1998-10-13', 3, true, 17),
-      (11, 'Ditto', '2022-5-14', 4, true, 22);
\ No newline at end of file
+      (11, 'Ditto', '2022-5-14', 4, true, 22);
+
+INSERT INTO owners (full_name, age) VALUES
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) Values
+    ('Pokemon'),
+    ('Digimon');
```

Comment:
Add these to your data.sql file
animals table should be modified to meet these requirements
1. if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead.
2. all animals got assigned owners

Reply:
Thanks!